src/picongpu/include/fields/laserProfiles/laserWavepacket.hpp

```diff
@@ -72,23 +72,22 @@ HINLINE float3_X laserLongitudinal(uint32_t currentStep, float_X& phase)
         envelope *= math::exp(-0.5 * exponent * exponent);
     }
 
+    phase = float_X(w * runTime + LASER_PHASE);
```

Comment:
shouldn't that be a `+=` ?
you are going to overwrite the input parameter by that (even if it is zero by default).